data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg

```diff
@@ -47,32 +47,40 @@
+
+Even merchants were rarely seen. Caravans between Aldril and Dan’Tonk would sometimes send a wagon to the village when called by the beacon tower."
+            {TB_BIGMAP}
+            {JOURNEY_REST_BEFORE_STAGE1}
         [/part]
         [part]
-            story=_ "Skeletons and zombies killed cattle and fired fields. <i>“Fear and obey Mordak the Mage!”</i> they cried in fell voices as they did their foul deeds. People vanished from isolated farmsteads. Men and women began to fear the night, and their children even the bright day. But the nearest lord was more than a day’s ride distant, and messengers sent to seek his help did not return."
+            story=_ "Then came the day that a dark mage settled in the region and began seeking sacrifices for his evil summonings.
+
+Skeletons and zombies killed cattle and fired fields. <i>“Fear and obey Mordak the Mage!”</i> they cried in fell voices as they did their foul deeds. People vanished from isolated farmsteads. Men and women began to fear the night, and their children even the bright day. But the nearest lord was more than a day’s ride distant, and messengers sent to seek his help did not return."
             background=story/Two_Brothers_M1P1.webp
         [/part]
         [part]
             story=_ "There was a man named Baran who had shown talent as a mage when he was young, gone to the great Academy on the Isle of Alduin, and returned to work his magic in the land where he was born. The people looked to him for help and leadership. He found weapons half-forgotten from the times of their sires and grandsires hanging in many houses, and bade the villagers to take them down and clean and oil them. He set the smiths of Maghre to making spearheads and ax-blades for the rest."
-            background=story/Two_Brothers_M1P1.webp
+            # Although I'd prefer to have less image changes during the story, this text fits neither the previous image of a necromancer, nor the next one which shows Arvith.
+            background=portraits/baran.webp
         [/part]
         [part]
-            story=_ "Now Baran had a brother named Arvith who had also left Maghre to seek his fortune, and had become the leader of a small band of horsemen who hired out as guards to merchant caravans. Fortunate it was for all that when Baran was but an apprentice mage, he had made a pair of amulets for himself and his brother, with which they might call to each other when in dire need. Baran sent out that call."
+            story=_ "Now Baran had a brother named Arvith who had also left Maghre to seek his fortune, and had become the leader of a small band of horsemen who hired out as guards to merchant caravans. Fortunate it was for all that, in better times, the brothers had talked about trade beacons and signals to use when in dire need. Baran sent out that call."
```

Comment:
```suggestion
            story=_ "Now Baran had a brother named Arvith who had also left Maghre to seek his fortune, and had become the leader of a small band of horsemen who hired out as guards to merchant caravans. Fortunate it was for all that, in better times, the brothers had talked about beacons and signals that might be used when in dire need. Baran sent out that call."
```

Reply:
Here I'm wanting to convey that the beacon was only expected to be used for trade, and it's something special to the brothers that they'd agreed an additional signal.

Reply:
How about the other version I gave you below?